### PR TITLE
Close socket descriptors on error

### PR DESCRIPTION
**Pull request by jswirl**
Close STUN datagram socket on error, also close blackhole socket on ICE deinit.

### CODE REVIEW

File: ice.c [hunk 1]
```diff
@@ -686,6 +687,8 @@ void janus_ice_deinit(void) {
 	janus_mutex_lock(&old_handles_mutex);
 	if(old_handles != NULL)
 		g_hash_table_destroy(old_handles);
+	if(janus_force_rtcpmux_blackhole_fd >= 0)
+	    close(janus_force_rtcpmux_blackhole_fd);
```
Comment (lminiero) [inline ice.c]: Use tabs and not spaces for indenting.

**Comment by lminiero** [comment]
Thanks for the patch. Before we can merge this, can you sign our [CLA](https://cla.conf.meetecho.com/janus/)?

**Comment by jswirl** [comment]
@lminiero Okay, done.

**Comment by lminiero** [comment]
Thanks, merging!

**Pull request merged**